src/main/resources/ccclubmanagement-firebase-adminsdk-h8s4n-bfd52e02b9.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "ccclubmanagement",
+  "private_key_id": "bfd52e02b9b61a115f80cc84580805bc4347eb61",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDHckZ+QoRvoSfo\nib4nloCU5P7kUhn6KkIa1Ua0ojDuCpxkArmdCjCooI9PJhFQCXEijxnsF2er7NI6\nPl4TO7Fqvuic6G0OYt4cTuVRBrpomx5yF+Sob53eE3Uc6z0ewdWTtNyhqBZ/xv98\n0CQm0VV/qwu63tNmgFL3h2kOZ0Tqs4w7K9kamBxnJnC+ZFK1VECyP09gnxBprutT\nAN9NU/sCYgUyKNAsfejJWKcnaYgErxKG96kSIxz3hBT7GsT2xgajcfV6X/DM1AbQ\nSo3QPR4xp2OTWTQxP53AhwF2I+ZFGRGXzh1XznMixI/jtoWhJ0y961XWjeNyqnCg\nFQDoOQMxAgMBAAECggEAAUqd9xxFYY71Fuypq3GJKCmJUIvJCZFCAAikcxD0UGHm\nZptSfcGPLzLt7bsZQstFX72TYat0a0GhJP5XZJtcXMJhWt36GkoNP6gpYvnh1qSy\nHC1gjzvA4+UZjaKQIGKFWnog5ucC8kkH/yrr/zNBIpbvjLf7uVNvK8b/B/IeoW4Z\ngHSj+kXitLcnBRoTBhspSRFT1mBvYBsRg5aESLagQoX8TaUcIDfVS+aZ62h849rl\n8XGn5w0/hdNMdd82oRfQRD76gmRuROzs495UkdRCP3gVMUQuuXUoBZ5wnH907qpe\nHcfB4FJ7AyAmjcWXKjRMHnEpNdn+uJJ9Ff0r3f4/6wKBgQDtukk8krGml7kRaq/v\nFrJdq/gruVBrq5QycJSbEaZtXdwO+D/94rXjPD7NqTxt9/s5+u8YwYOZ+LcUsOD3\ni6aZN6ymcYRkArhvu5n628fSwvLV8JJFyF5crsk1FGB9QVn/xcgBrycU355mpzRE\nGcjq37aBFLvOTkT/raLLZJltswKBgQDWxryoLlAtWaB1JzPDeb3+Uwk6RNLQ+ozo\nGyN/ONym3BQDlZPOA3qwvdc4sWketSfAe7IhJND7R28pA120XUNdx0v4a5U0HIwo\neH3A/NAB6b45eg46FMUUT80rMnb58ZM5ACGx3iCfix6IIQNUk7WZfqCc9YR7mtE9\nPzHu87BBiwKBgG1yAh2c+6TS3SUjlPPh4yQ09ABNRLUNPSErq37o1VIskA9kQzHD\nmjGPnupYK7YFIOhEmqcvEENmugMQm8HbzUtk0luej3Drv7XRDMbtQEXL8riEFROD\nd837ui5xvyprV9/lUntUBA4cFXT1LZ7csB9R91feD8ifSCzd58D0A575AoGAanm4\nCYB8Sx9Fxw6aQOxX3n938aUxBl1QAbmCjY25BhfQ7D35OobLClfzHK4HOSwB+xDX\nrEq+N/SgDSsYB1S9Ev68kPl0zJLigSXK4iPzUx94KLJz3yNw2LL/Oc0Y4qx6miWQ\njsdaUP39Mut65TEdNG1sXqPw+/4ygTSVIqs41CUCgYBbbsmOw0n9EH2OjE5R7FYu\ng3OaiwF/4/zNcidcPkErfo2DzpgWVTNczi0Z7qK/bDUdTjfIE/o5hfUkvBrg74fx\nzI1yKFDwk1sQY3cIwkLymZQAbMNmPScyVO0SdKDKKDiCCo+eyyW5ChHivaMRU0jV\n0bx/Vfg7WhJP7KPud37avQ==\n-----END PRIVATE KEY-----\n",
+  "client_email": "firebase-adminsdk-h8s4n@ccclubmanagement.iam.gserviceaccount.com",
+  "client_id": "101773667337515324592",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-h8s4n%40ccclubmanagement.iam.gserviceaccount.com"
+}
```

Comment:
Ok to commit?

Reply:
fixed in a490ba3231089e79dfdb813c62d57670f092d289